packages/tools/replay-tool/src/replayFluidFactories.ts

```diff
@@ -68,6 +72,27 @@ export class ReplayRuntimeFactory extends RuntimeFactoryHelper {
     }
 }
 
+export const nonComparableDdsFactories: IChannelFactory[] = [
```

Comment:
For these 4 channels, we want to do some minimal comparison (right now only .attributes blob)? Can you please mention in a comment what not comparable here means and why.